neon_local/src/main.rs

```diff
@@ -540,6 +540,33 @@ fn handle_tenant(tenant_match: &ArgMatches, env: &mut local_env::LocalEnv) -> an
                 "tenant {} successfully created on the pageserver",
                 new_tenant_id
             );
+
+            // Create an initial timeline for the new tenant
+            let new_timeline_id = parse_timeline_id(create_match)?;
+            let timeline = pageserver
+                .timeline_create(new_tenant_id, new_timeline_id, None, None)?
+                .ok_or_else(|| {
+                    anyhow!(
+                        "Failed to create initial timeline for tenant {}",
+                        new_tenant_id
+                    )
+                })?;
+            let new_timeline_id = timeline.timeline_id;
+            let last_record_lsn = timeline
+                .local
+                .expect("no local timeline info")
```

Comment:
> `.expect`

Let's avoid panics if we can, `.context` or `.with_context` + ids would be safer: panic kills the entire thread and [exits](https://neondb.slack.com/archives/C036U0GRMRB/p1652201272837079?thread_ts=1652185280.790199&cid=C036U0GRMRB) the pageserver, which is bad.